Keas.Mvc/ClientApp/components/Access/RevokeAccess.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Alert, Button } from 'reactstrap';
+import {  IAccessAssignment } from '../../Types';
+import AccessModal from './AccessModal';
+
+interface IProps {
+  assignment?: IAccessAssignment;
+  revoke: (accessAssignment: IAccessAssignment) => Promise<void>;
+  cancelRevoke();
+}
+
+interface IState {
+  submitting: boolean;
+  error?: string
+}
+
+export default class RevokeAccess extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+
+    this.state = {submitting: false}
+  }
+  public render() {
+    const valid = !!this.props.assignment
+    const assignment = this.props.assignment
+    return (
+      valid ? 
+      <AccessModal
+        isOpen={true}
+        closeModal={this.props.cancelRevoke}
+        header={<h1>Revoke Access for {this.props.assignment.person.firstName}</h1>}
+        footer={
+          <Button
+            color='primary'
+            onClick={() => {
+              this.setState({submitting: true})
+              this.props.revoke(this.props.assignment).catch(e => {
+                this.setState({
+                  error: "Error -- " + e.message + " -- Please try again later",
+                  submitting: false,
+                })
+              })
+            }}
```

Comment:
@klanmiko i'd prefer this be its own function instead, and following the same pattern as the other revokes